services/sprintBacklog.js

```diff
@@ -49,6 +50,29 @@ const getSprintBacklog = async (sprint, totalSpentHoursBugs) => {
 	return response;
 };
 
+/*
+	TODO: understand what we will have to put on the sheet DB
+	because it is not the same values that we have as the base spreadsheet that we are referring to
+*/
+const updateDBWithStoryDetails = async ({
+	sprint,
+	bugsData,
+	sprintIndicators,
+}) => {
+	const sprintSheetName = `Sprint-${sprint}`;
+	try {
+		const spreadSheet = new GoogleSpreadsheet(process.env.GOOGLE_SHEET_ID);
+		await spreadSheet.useServiceAccountAuth({
+			client_email: process.env.GOOGLE_SERVICE_ACCOUNT_EMAIL,
```

Comment:
might be worthy to merge @lucasbald code and add it on ssm later

Reply:
done